src/plots/cartesian/select.js

```diff
@@ -45,6 +47,7 @@ module.exports = function prepSelect(e, startX, startY, dragOptions, mode) {
     outlines.enter()
         .append('path')
         .attr('class', function(d) { return 'select-outline select-outline-' + d; })
+        .attr('transform','translate(' + xs + ' ' + ys + ')')
```

Comment:
Could you add a `,` between the `translate` values? I know it's not strictly necessary but we use them nearly everywhere else in the codebase.